string shape
string color
float32 score
int32[] bbox